src/Utils/normalization.jl

```diff
@@ -0,0 +1,85 @@
+"""
+    normalize(system)
+
+Normalize a continuous system.
+
+### Input
+
+- `system` -- continuous system
+
+### Output
+
+Either the same system if it already conforms to our required structure, or a
+new system otherwise.
+
+### Algorithm
+
+We apply the following normalization steps.
+
+* [`normalize_inputs`](@ref)
+"""
+function normalize(system::InitialValueProblem{<:Union{AbstractContinuousSystem,
+                                                       AbstractDiscreteSystem}})
+    return normalize_inputs(system)
+end
+
+"""
+    normalize_inputs(system)
+
+Normalize the inputs of a continuous system.
+
+### Input
+
+- `system` -- continuous system
+
+### Output
+
+Either the same system if the inputs are of type `AbstractInput`, or a new
+system that wraps the inputs in a `ConstantInput`.
+"""
+function normalize_inputs(system)
+    inputdim(system) == 0 && return system
+    U = inputset(system)
+    U isa AbstractInput && return system
+    if !(U isa LazySet)
+        throw(ArgumentError("inputs of type $(typeof(U)) are not supported"))
+    end
+    return IVP(wrap_inputs(system.s, U), system.x0)
+end
+
+function wrap_inputs(system::CLCCS, U::LazySet)
+    return CLCCS(system.A, system.B, system.X, ConstantInput(U))
+end
+
+function wrap_inputs(system::CLCDS, U::LazySet)
+    return CLCDS(system.A, system.B, system.X, ConstantInput(U))
+end
+
+function wrap_inputs(system::CACCS, U::LazySet)
```

Comment:
Note that `BFFPSV` will ignore the `system.c` (even if it accepts this type, so it can be considered a bug). One way is to annotate the type of systems that can be handled by `BFFPSV`, unrelated to this file (i prefer that). 

Another way is to normalize that `CACCS` into a `CLCCS` here in such a way that the input is `U + Singleton(c)` or even `U + c` (a lazy translation).

Reply:
I've opened #527 as a follow up.

Reply:
Yes, I took a private note of this, but I should have just opened an issue :+1: